js/navigation/routes.js

```diff
@@ -2,13 +2,15 @@ const ROUTES = {
   // Ingress
   INGRESS: 'INGRESS',
 
-  // Unhautenticated
-  LOGIN: 'LOGIN',
+  // Authentication
+  AUTHENTICATION: 'AUTHENTICATION',
+  AUTHENTICATION_LANDING: 'AUTHENTICATION/LANDING',
```

Comment:
is it the plugin that requires the `/` ? can't we use the same identifiers as we do for `INGRESS` ?